ci/push_chart.sh

```diff
@@ -10,11 +10,17 @@ source "$CURR_DIR/common.sh"
 
 #########################################################################################
 
-if [ -z "$TRAVIS_TAG" ]  ; then
-  info "No TRAVIS_TAG in environment: no Helm package will be built..."
+# Check for update to version of Chart.yaml
+
+version_changed=$(git diff $TRAVIS_COMMIT_RANGE Chart.yaml | grep +version)
+
+if [[ -z $version_changed ]]
+then
```

Comment:
This would be better written as

```
if ! git diff $TRAVIS_COMMIT_RANGE Chart.yaml | grep -q +version; then
```

Reply:
Ahah! The one liner I was looking for! Thanks!